modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -4,24 +4,24 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint platform, permissions, configuring, accounts
 
-This topic describes how to create and manage your Anypoint Platform account.
+Create and manage your Anypoint Platform account.
 
-== To Create an Anypoint Platform Account
+== Create an Anypoint Platform Account
 
 [NOTE]
 If you already have a Mule Community account you can use that account to access Anypoint Platform.
 
 . Go to https://anypoint.mulesoft.com/#/signup[Signup page].
```

Comment:
the signup page isn't a product and that label "Signup page" doesn't exist on the page, so I'd change to "Go to the https://anypoint.mulesoft.com/#/signup[signup page]."